tools/slang-unit-test/unit-test-digest-utils.cpp

```diff
@@ -29,4 +29,24 @@ SLANG_UNIT_TEST(checksum)
         String testCString = hashToString(testC);
         SLANG_CHECK(testCString.equals(String("11111111222222223333333344444444")));
     }
+
+    {
+        String digestString = String("5D6CC58E1824A4DFD0CF57395B603316");
+        slang::Digest digest = stringToHash(digestString);
+
+        SLANG_CHECK(digest.values[0] == 2395303005);
+        SLANG_CHECK(digest.values[1] == 3752076312);
+        SLANG_CHECK(digest.values[2] == 962056144);
+        SLANG_CHECK(digest.values[3] == 372465755);
```

Comment:
It is better to just convert digest back to string here and check the string is equal to `digestString`.